src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+const notification = document.querySelector('[data-qa="notification"]');
+
+const firstPromise = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    clearTimeout(timeoutId);
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+
+  document.addEventListener('click', function handleClick(e) {
+    if (e.button === 0) {
+      document.removeEventListener('click', handleClick);
+      clearTimeout(timeoutId);
+      resolve('First promise was resolved');
+    }
+  });
+});
+
+firstPromise.then((message) => {
+  notification.classList.add('success');
+  notification.textContent = message;
+}).catch((error) => {
+  notification.classList.add('warning');
+  notification.textContent = error.message;
```

Comment:
Create a separate function for showing a notification as the logic is almost the same for all promises